ocaml/xapi/xapi_vtpm.ml

```diff
@@ -12,6 +12,30 @@
    GNU Lesser General Public License for more details.
  *)
 
+(** Don't allow unless VTPM is enabled as an experimental feature *)
+let assert_not_restricted ~__context =
+  let pool = Helpers.get_pool ~__context in
+  let restrictions = Db.Pool.get_restrictions ~__context ~self:pool in
+  let feature = "restrict_vtpm" in
```

Comment:
This is probably fine while it is temporary, but once it is not experimental anymore we should remove the restriction from here so this also works on non-Citrix builds (or add `vtpm` to the list of default features in Features.mli.
vtpm is expected to be a feature that is always available (once we finish implementing it...)

So perhaps add a comment here to remind ourselves to remove this.

Reply:
Creating an issue so we don't forget about this